modules/react/common/stories/examples/ResponsiveStyles.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import {Box, Grid} from '@workday/canvas-kit-react/layout';
+import styled from '@emotion/styled';
+import {type} from '@workday/canvas-kit-react/tokens';
+import {
+  useResponsiveContainerStyles,
+  useResizeObserver,
+  ResponsiveContextProvider,
+} from '@workday/canvas-kit-react/common';
+import {FormField} from '@workday/canvas-kit-react/form-field';
+import {Select, SelectOption} from '@workday/canvas-kit-react/select';
+
+const StyledHeading = styled(Box.as('h3'))({
+  ...type.levels.body.large,
+  ...type.variants.inverse,
+  margin: 0,
+  fontWeight: type.properties.fontWeights.bold,
+});
```

Comment:
You can use `Text` component for this instead of `StyledHeading`.

Reply:
Updated